runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -159,9 +159,9 @@ static atomic_uint_least32_t next_thread_idx;
 
 #define CHPL_INTERNAL_ERROR(msg)                                        \
         do {                                                            \
-          DBG_P_LP(1, "%s:%d: internal error: %s",                      \
+          DBG_P_LP(~0U, "%s:%d: internal error: %s",                    \
                    __FILE__, (int) __LINE__, msg);                      \
-          fflush(debug_file);                                           \
+          fflush(NULL);                                                 \
```

Comment:
The original code only flushed the debug output file before aborting. The new code flushes all open files. I changed this because I had configured uGNI to produce debug output into node-specific files also, but those weren't getting flushed before the `abort()` and I was losing the last bit of output sent to them because it was still buffered in `stdio`.

Reply:
Gotcha. And the `1` -> `~0U` change?

Reply:
Ah. That was because using `1` there meant that the message was only actually printed if the user had set the `0x1` bit in the debug flag (`DBGF_IFACE`). I was running with the debug flag set to `0x10010` and not seeing the internal error message. So I changed the value from `1` to `~0U` (should actually have been `~0UL`) so it would get printed with _any_ debug flag value set. However, there's still a problem, as I discovered while hunting for the `miniUnorderedCopyStress` problem just now: if you build with comm debug on but run without any debug flags set at all, you still don't set the output! So I'm going to change this to print to `stderr` directly, no matter where the debug output is being sent.